assets/stylesheets/common/rowcolumn.scss

```diff
@@ -0,0 +1,55 @@
+.bb-row {
+  width: 100%;
+  clear: both;
+  overflow: hidden;
+  margin: 0;
+  &[data-span="column-width-span1"] {
```

Comment:
correct me if i'm wrong; I think this style only applies to `<div class="bb-row" data-span="column-width-span1">` and not to `.bb-column`?

Reply:
<div class="bb-row">
<div class="bb-column" data...>
</div>
</div>

Would the above not mean that first .row then .row .colum? I haven't really used class within class much in a long while till this repo, but without column it didn't look right when previewed.

Reply:
If I remember correctly

```scss
.class-name {
  &[attr="val"] {
    ...
  }
}
```
translates into 
```scss
.class-name[attr="val"] {
  ...
}
```

Reply:
Hmmm, I'll try it out and see if I can knock another off the white-list.

Reply:
With div b-rowrow, div bb-column class: 
![image](https://github.com/RpNation/bbcode/assets/55112994/f60befb3-abd7-4f49-a64a-31d8e2d8fa0e)

Without div bb-column and data class: 
![image](https://github.com/RpNation/bbcode/assets/55112994/ba3869c1-1bf8-4ff0-9b6d-2cb621f0ff80)

With div bb-row and div bb-row and data: 
![image](https://github.com/RpNation/bbcode/assets/55112994/f097bf81-9e6a-4bc0-a937-55fbe62cffda)

Reply:
the CSS you currently have doesn't work
<img width="858" alt="image" src="https://github.com/RpNation/bbcode/assets/42795314/c428e4d8-bee6-486c-9a33-927a6b1f23b9">